src/metpy/io/gini.py

```diff
@@ -19,7 +19,10 @@
 try:
     from xarray import Variable
     from xarray.backends.common import AbstractDataStore
-    from xarray.core.utils import FrozenOrderedDict
+    try:
+        from xarray.core.utils import FrozenOrderedDict as FrozenDict
```

Comment:
I think I'd prefer this to try to import `FrozenDict` initially, and then fall back to `FrozenOrderedDict`. To me that makes more logical sense, to try the main, future-proof path, and then fall back to the old way.

Reply:
Sounds good - didn't know which way this should work generally. I'll update.

Reply:
Yeah, I'm not sure about generally. But it makes sense to me to put in the `except` clause the code that should be deleted when we require a new enough xarray.